docs/docs/getting-started/troubleshooting.md

```diff
@@ -0,0 +1,31 @@
+# Troubleshooting
+
+## Known issues
+
+- QueryException "AcquireFailurePointNotOnChain"
+
+    + Description: It ocurs when you attempt to start a head using a point in the past too old (exceeding **k** limit).
+
+    + Reference: [439](https://github.com/input-output-hk/hydra/issues/439) AcquireFailurePointTooOld when `--start-chain-from` point is past **k**
+
+    + Workaround: Restart your node fresh, without state. For that you need to remove your persistance dir and restart the hydra-node.
+
+- Hydra node crashes after a fork
+
+    + Description: It ocurs during a fork and expects the operator to restart its hydra-node.
+
+    + Reference: [560](https://github.com/input-output-hk/hydra/issues/560) Hydra node crashed after a fork
+    
+    + Workaround: Restarting your node should be enough to come back to live. Beware, if you wait to long to restart it then you may fall under `QueryAcquireException AcquireFailurePointTooOld` and will require you to restart without state.
+
+- Head can collect at most 3 commits and each party can commit either 1 or 0 UTXO to a Head. Only no or one utxo can be committed to a Head.
```

Comment:
3 is not the latest limit. I think we allow heads of 4 participants. That means we can also have 4 commits.

I think the known issue here is rather that we only allow one utxo (or none) per participant.

Reply:
Can you keep it non-specific by not mentioning a number? People should check https://hydra.family/head-protocol/benchmarks/transaction-cost/ for maximum number of participants.

Reply:
@ch1bo in the release notes this known issue is mentioned but with different values: `9` and latest `3`

will use the reference instead! ty!